doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -617,7 +617,7 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``b96453f653b674629c617bdbc810c3d9f7916ef4``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``cd16a8388f71a6cce0cea871f75f6d4ac8f56da9``, plus some |NCS| specific additions.
```

Comment:
The Zephyr SHA should be also applied in the following part: https://github.com/nrfconnect/sdk-nrf/blob/b75b91188dbc0ecef7d40ce07c9786bec6130c17/doc/nrf/releases/release-notes-changelog.rst#L626-L634